pkgs/development/libraries/ubdsrv/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkg-config, liburing }:
+
+stdenv.mkDerivation rec {
+  pname = "ubdsrv";
+  version = "1.1-rc1";
+
+  src = fetchFromGitHub {
+    owner = "ming1";
+    repo = "ubdsrv";
+    rev = "v${version}";
+    sha256 = "sha256-xY1iWsZKPnri1eZV/L1LWMkNtPERGofoZ1262DdcJWg=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+  buildInputs = [ liburing ];
+
+  doCheck = true;
+
+  meta = with lib; {
+    description = "Userspace block device driver";
+    homepage = "https://github.com/ming1/ubdsrv";
+    platforms = platforms.linux;
+    maintainers = [ maintainers.lx ];
```

Comment:
Since it is a dependency of a package I maintain, can you list me as maintainer too?